.github/workflows/code-formatting.yml

```diff
@@ -74,3 +74,23 @@ jobs:
       - name: mypy check
         run: |
           mypy
+
+  dead-code-vulture:
```

Comment:
if it fails, will it block merging a PR?
@Borda

Reply:
Shouldn't do. Comes up as a serperate check that isn't required for merge.

Reply:
No, it is not required check